hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -152,22 +152,21 @@ def copy(self, default=None):
         return new
 
     def _sync_user(self, user, employee_has_image=False):
+        """Set calendar_ids from 'Create employee' button from user and set date_start
+        as today is totally correct in a real use case.
+        On the other hand, in test use cases it is problematic to define today in
+        date_start because fixed dates are frequently used.
```

Comment:
If this is the problem, you have to use `freezetime` from freezegun library instead of doing other weird things.